src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstBase;
+    private int secondBase;
+    private int height;
+
+    public IsoscelesTrapezoid(Color color, int firstBase, int secondBase, int height) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
```

Comment:
It's a good practice to avoid passing expressions directly into the constructor. Consider creating separate variables for `firstBase`, `secondBase`, and `height` before passing them to the constructor to enhance readability.